src/LM/api.jl

```diff
@@ -0,0 +1,23 @@
+#TO DO 
+# Doc string
+function maskedscore(m::Langmodel,temp_lm::DefaultDict,word,context)
+   score(m,temp_lm,lookup(m.vocab ,[word])[1] ,lookup(m.vocab ,[context])[1])
```

Comment:
```suggestion
   score(m, temp_lm, lookup(m.vocab, [word])[1], lookup(m.vocab, [context])[1])
```
This is just a suggestion. I know it will be a lot of work making code style changes in the entire PR, so feel to incorporate it or not.